lib/librato/collector/aggregator.rb

```diff
@@ -89,18 +92,22 @@ def measure(*args, &block)
         if args.length > 1 and args[-1].respond_to?(:each)
           options = args[-1]
         end
-        source = options[:source]
+
+        if options[:source]
```

Comment:
Re-added this for custom instrumentation. Scenario where user has upgraded to librato-rack MD, but still has custom instrumentation using legacy source:

```ruby
Librato.increment 'user.purchases', source: user.id
```

Since the user has already upgraded to MD, we should still allow source arg but submit as a tagged measurement, e.g. ```{ source: user.id }```